packages/state-transition/test/unit-mainnet/sanityCheck.test.ts

```diff
@@ -4,7 +4,7 @@ import {beforeProcessEpoch} from "../../src/index.js";
 import {generatePerfTestCachedStateAltair, generatePerfTestCachedStatePhase0, perfStateId} from "../perf/util.js";
 
 describe("Perf test sanity check", () => {
-  vi.setConfig({testTimeout: 60 * 1000});
+  vi.setConfig({testTimeout: 90 * 1000});
```

Comment:
Why did this value change?  Seems unrelated to the runner changing...

Reply:
Not necessary, but faced random timeout on this test multiple times during testing, so make sure CI stay stable increased this timeout.